specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -4581,6 +4586,96 @@
       },
       "description": "An error response for a resource management request."
     },
+    "ApiProfile": {
+      "properties": {
+        "profileVersion": {
+          "type": "string",
+          "description": "The profile version."
+        },
+        "apiVersion": {
+          "type": "string",
+          "description": "The API version."
+        }
+      }
+    },
+    "AliasPathMetadata": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "NotSpecified",
+            "Any",
+            "String",
+            "Object",
+            "Array",
+            "Integer",
+            "Number",
+            "Boolean"
+          ],
+          "x-ms-enum": {
+            "name": "AliasPathTokenType",
+            "modelAsString": false,
```

Comment:
this should be true unless you have an explicit need to model this as a native numeric enum in all SDK languages